deploy/terraform/ldapi/modules/server/main.tf

```diff
@@ -32,6 +50,13 @@ resource "google_project_iam_member" "ldapi_service_account_permissions" {
   member = "serviceAccount:${google_service_account.ldapi_service_account.email}"
 }
 
+resource "google_compute_disk" "datahost_ldapi_data" {
+  name = local.data_disk_name
+  type = "pd-ssd"
+  zone = var.zone
+  size = 10
```

Comment:
This seems reasonable.. I don't think we'll ever fill it. I have ptsd from things silently failing when disks fill up though.. is it possible to set up some sort of alert when it reaches like 90% capacity? Or auto-resize or something? Probably not worth worrying about I guess.. overall this is great.

Reply:
Maybe we could create a separate issue for configuring monitoring? We'll need it for multiple metrics in future.